app/jobs/web_hook_job.rb

```diff
@@ -10,7 +10,14 @@ def perform(procedure_id, dossier_id, state, updated_at)
       state: state,
       updated_at: updated_at
     }
+
     procedure = Procedure.find(procedure_id)
-    Typhoeus.post(procedure.web_hook_url, body: body, timeout: TIMEOUT)
+
+    Sentry.set_tags(procedure: procedure_id, dossier: dossier_id)
+    Sentry.set_extras(web_hook_url: procedure.web_hook_url)
+
+    response = Typhoeus.post(procedure.web_hook_url, body: body, timeout: TIMEOUT)
+
+    raise StandardError, response if !response.success?
```

Comment:
Ça va faire des retry en pagaille, non ? C'est ce que tu veux ? Pour moi c'est un breaking change.

Reply:
Ce n'est pas forcément ce que je veux. Ce que je souhaite, c'est avoir une trace quand la réponse du webhook n'est pas un succès. Plutôt que d'utiliser le mécanisme de Sentry, je pourrai plus simplement écrire quelque chose comme : 
`Rails.logger.error "[Webhook] Error for procedure#{procedure_id} dossier#{dossier_id}. Webhook response status : #{response.status} Message #{response.body}"

Qu'en penses-tu ?